Neos.ContentRepository.Core/Classes/Projection/ContentGraph/ContentGraphInterface.php

```diff
@@ -158,4 +148,8 @@ public function getDimensionSpacePointsOccupiedByChildNodeName(
      * @internal only for consumption in testcases
      */
     public function countNodes(): int;
```

Comment:
This one is a very odd duck now as it is only used for tests AND operates globally (so doesn't take the workspace this is bound to into account). I am open for suggestions. We could make it operate on the workspace / content stream and see if that works for the tests too?!

Reply:
Yeah or at the very least declare it as experimental and tell that it operates globally ... would be definitely fine to address this in a followup ;)

Reply:
Mmm, adapting it to operate on bound content stream is not so trivial, we then need to join on hierachy relation and I fear this muddles with the intend of this being super low level and catching possible mismatches in the projection as well 🤔

Reply:
Yes dont worry i think we can open a followup issue for this and discuss wether we like this and how we like this ;)
(For example it _could_ be a method directly on the `ContentGraphFinder`)